examples/kingworld/README.md

```diff
@@ -0,0 +1,15 @@
+# KingWorld Starter
+
+## Scaffold
+To get start with this template, simply paste the command in to your terminal:
+```bash
+bun create kingworld
+```
+
+## Development
+KingWorld is set with `nodemon` for auto reload, simply run:
+```bash
+bun run dev
+```
+
+Go to http://localhost:3000 and enjoy your new awesome web server
```

Comment:
Please use ``Open http://localhost:3000/ with your browser to see the result.`` like the other templates.